packages/eas-cli/src/devices/actions/__tests__/create-test.ts

```diff
@@ -18,6 +18,14 @@ beforeEach(() => {
 
 describe(DeviceCreateAction, () => {
   describe('#runAsync', () => {
+    let originalConsoleLog: () => void;
```

Comment:
I think that it cleaner to write this code with beforeAll/afterAll and in top level scope of this file.

It's easier to copy that to different tests and it's not mixing with test code